### DISCUSSION TITLE
Suppress Legacy Warnings

### DISCUSSION THREAD

**Discussion by scath999** [sqlalchemy/sqlalchemy#11550] [Usage Questions] [answered]
Hi There,

Love Sqlalchemy - thanks for creating this. 

That said, I'm getting a ridiculous number of LegacyAPI warnings as I iterate on a new project.

`LegacyAPIWarning: The Query.get() method is considered legacy as of the 1.x series of SQLAlchemy and becomes a legacy construct in 2.0. The method is now available as Session.get() (deprecated since: 2.0) (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
  return User.query.get(user_id)`

I completely understanding wanting to alert devs about deprecated calls  (of course!) but is there a way to suppress these warnings? They are really cluttering my console and making it hard to see important feedback.

Thanks!

**Comment by gordthompson** [comment, marked as answer]
I used this in my startup code and it worked for me:

```python
import warnings
from sqlalchemy import exc as sa_exc
warnings.simplefilter("default")
warnings.simplefilter("ignore", category=sa_exc.LegacyAPIWarning)
```

**Comment by scath999** [comment]
Thanks, Gord :)
